opentelemetry-sdk/src/lib.rs

```diff
@@ -146,6 +146,7 @@ pub mod trace;
 #[doc(hidden)]
 pub mod util;
 
-pub use instrumentation::{InstrumentationLibrary, Scope};
+#[doc(inline)]
+pub use opentelemetry::InstrumentationScope as Scope;
```

Comment:
I believe we can remove this aliasing. Or atleast not expose this to end-user, if we want to use it internally in sdk. I understand this is coming from instrumentation.rs, but is not necessary.

Reply:
done